stories/assets/js/parallax_lines.js

```diff
@@ -0,0 +1,25 @@
+import './viewport';
+import { breakpoints } from './global-variables';
+
+export function parallaxlines() {
+  $(window).scroll(() => {
+    var $sticky = $('.ourexpertise .glide__slides');
+    var stickyTop = $sticky.offset().top;
+    var windowTop = $(window).scrollTop();
+    var diff = windowTop + 500;
+    $('.ourexpertise .glide__slides').each(function () {
+      if ($(window).width() > breakpoints.small) {
+        $('.ourexpertise').css({ 'background-image': 'none', 'border-right': 'none', transition: 'none' });
```

Comment:
@pallavikadam-srijan Kindly use css classes to toggle effect. That way if there is color/css property change then we do it in css class and not inside Js.
Eg.:
```
$(".ourexpertise").addClass("foo");
```
or
```
$(".ourexpertise").removeClass("foo");
```
or
```
$(".ourexpertise").toggleClass("bar");
```